utils/ext2.c

```diff
@@ -0,0 +1,47 @@
+#include <myst/eraise.h>
+#include <myst/ext2.h>
+
+int myst_ext2_test(const char* path)
```

Comment:
A little ambiguous name. Suggest `myst_ext2fs_check`

Reply:
I backed out most of the changes to ``./tools/myst/package.c`` so this is no longer needed now that we don't support these:

```
$ myst package <ext2> ...
$ msg package <cpio> ...
```

It didn't make sense to allow ext2 as an argument but then only use it to extract the roothash. This is very odd since the ``--roothash`` option can be repeated (and ``myst`` could not extract the public key from the ext2 image anyway).